woocommerce/woocommerce
# Do not remove "sale date from" when the sale is still active

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Do not delete "sale date from" when the sale is still active, as in some legislations it's mandatory to show the start and end date of sales, and developers can get it from this field.

Closes #34696 

### How to test the changes in this Pull Request:

1. Use a REST client to update a product to set it on sale in 5 minutes: (Edit timestamps and price as needed)
```
PUT /wp-json/wc/v3/products/PRODUCTID

{
"sale_price": "30",
"date_on_sale_from":"2023-10-16T12:15:00+03:00",
"date_on_sale_to":"2023-10-18T11:15:00+03:00"
}
```

2. A few minutes after the sale starts check the product and start date should not be blank

<details>

#### Significance


#### Type




</details>
